src/main/kotlin/org/rucca/cheese/model/NotificationsGet200ResponseDataPageDTO.kt

```
@@ -0,0 +1,33 @@
+    @get:JsonProperty("has_prev")
+    val hasPrev: kotlin.Boolean? = false,
+    @Schema(example = "null", description = "")
+    @get:JsonProperty("prev_start")
+    val prevStart: kotlin.Int? = 0,
+    @Schema(example = "null", description = "")
+    @get:JsonProperty("has_more")
+    val hasMore: kotlin.Boolean? = true,
+    @Schema(example = "null", description = "")
+    @get:JsonProperty("next_start")
+    val nextStart: kotlin.Int? = 7020
+) {}
```

detekt (1.23.7)

[warning] The class or object NotificationsGet200ResponseDataPageDTO is empty.

(detekt.empty-blocks.EmptyClassBlock)